### ISSUE TITLE
Allow audit to player's max carry weight through game_balance.json

### ISSUE THREAD

**Issue by shadowonwhite** [CleverRaven/Cataclysm-DDA#74220]
### Is your feature request related to a problem? Please describe.

A default character with 8 strength can comfortably carry 100lb without stamina or movement penalty, something I wish to be different.

### Solution you would like.

Allow audit to player's max carry weight through game_balance.json like stamina and heal rate.

### Describe alternatives you have considered.

Way to apply certain mutations on all players, NPCs, on spawn, easily.

### Additional context

100lb of water for context

![water](https://github.com/CleverRaven/Cataclysm-DDA/assets/69741262/bbceb4ec-3ba5-42a9-acfd-dd0ac8b0b4af)

**Comment by RenechCDDA** [comment]
No reason to make this an external option, make a mod if you want to do this.

**Comment by shadowonwhite** [comment]
I think did do it through custom mutation, yet the problems being i have to remember to select it at the beginning of every run, make sure all NPCs spawn with it by auditing NPC_starting_traits (not sure if modded file will completely replace stuff in "[]"), and have an awkward trait to look at in everyone's stat menu.

**Comment by Hyperseeker** [comment]
For what it's worth, I tried modelling carry weight's effect on stamina in [the mod called Weight](https://github.com/FrontierModsExtra/Weight). It's active by default, though only for your current character (i.e., not for NPCs). I believe it should work for being added mid-playthrough (I used the EOC event that specifically activates on new games and loaded ones), though I haven't tested it myself.

I haven't considered making it adjustable, especially through `game_balance.json` settings, but it might be possible now that I'm considering a mod menu.
